### ISSUE TITLE
Hypertrichotic hyperpigmented patch

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6233]
**Preferred term label:**
Hypertrichotic hyperpigmented patch

**Synonyms**
Cutaneous hyperpigmentation with overlying hypertrichosis

**Definition (free text, please give PubMed ID)**
A large nonpalpable spot of the skin over 1 cm in dimension with increased pigmentation and increased hair growth.
PMID: 18410979
**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0000953

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
602782   PMID: 18410979 (10/10)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
also PMID: 19175903, 2/2

**Comment by pnrobinson** [comment]
added as HP_0033190
